msnc/model.py

```diff
@@ -64,21 +64,21 @@ def run_training(
         self,
         output_dir,
         training_set,
+        development_set,
         epoch_num=100,
         checkpoint_interval=10,
-        development_set=None,
         save_best_model=True,
     ):
         """Run training procedure
 
         Arguments:
             output_dir_path {str} -- path to output dir
             TODO training_set {} -- dataset for training
```

Comment:
Ah, it's time to annotate types of training_set and development_set.
It should be `List[??]`.

Reply:
Is this correct? 04061ae

Reply:
Sorry, you're right.